Edabit/Medium/Days in a Month.js

```diff
@@ -0,0 +1,4 @@
+// Problem link: https://edabit.com/challenge/wNhCHwPXNLJwrTith
```

Comment:
Where is `"use strict" in this file.
Use that for every file.
Also if possible resubmit them in `Edabit`